content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

```diff
@@ -125,12 +131,22 @@ Cert-manager has added [experimental Support for Kubernetes `CertificateSigningR
     $ kubectl apply -f ./selfsigned-issuer.yaml
     {{< /text >}}
 
+## Verify secrets are created for each cluster issuer
+
+{{< text bash >}}
+$ kubectl get secret -n cert-manager -l controller.cert-manager.io/fao=true
+NAME                  TYPE                DATA   AGE
+bar-ca-selfsigned     kubernetes.io/tls   3      3m36s
+foo-ca-selfsigned     kubernetes.io/tls   3      3m36s
+istio-ca-selfsigned   kubernetes.io/tls   3      3m38s
+{{< /text >}}
+
 ## Export root certificates for each cluster issuer
 
 {{< text bash >}}
-$ export istioca=$(kubectl get clusterissuers istio-system -o jsonpath='{.spec.ca.secretName}' | xargs kubectl get secret -n cert-manager -o jsonpath='{.data.ca\.crt}' | base64 -d)
-$ export fooca=$(kubectl get clusterissuers foo -o jsonpath='{.spec.ca.secretName}' | xargs kubectl get secret -n cert-manager -o jsonpath='{.data.ca\.crt}' | base64 -d)
-$ export barca=$(kubectl get clusterissuers bar -o jsonpath='{.spec.ca.secretName}' | xargs kubectl get secret -n cert-manager -o jsonpath='{.data.ca\.crt}' | base64 -d)
+$ export istioca=$(kubectl get clusterissuers istio-system -o jsonpath='{.spec.ca.secretName}' | xargs kubectl get secret -n cert-manager -o jsonpath='{.data.ca\.crt}' | base64 -d | sed 's/^/         /')
+$ export fooca=$(kubectl get clusterissuers foo -o jsonpath='{.spec.ca.secretName}' | xargs kubectl get secret -n cert-manager -o jsonpath='{.data.ca\.crt}' | base64 -d | sed 's/^/         /')
+$ export barca=$(kubectl get clusterissuers bar -o jsonpath='{.spec.ca.secretName}' | xargs kubectl get secret -n cert-manager -o jsonpath='{.data.ca\.crt}' | base64 -d | sed 's/^/         /')
```

Comment:
```suggestion
$ export barca=$(kubectl get clusterissuers bar -o jsonpath='{.spec.ca.secretName}' | xargs kubectl get secret -n cert-manager -o jsonpath='{.data.ca\.crt}' | base64 -d | sed 's/^/      /')
```

Reply:
@frankbu This does not work, 8 spaces are needed, for istioctl to accept the config. Tested it out locally. I have taken care of other comments.

Reply:
I'm surprised because it looks like 6 spaces is the current indentation. Anyway, thanks for checking!